examples/user/grid_search_params.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from dotenv import load_dotenv
+
+from genai.model import Credentials, Model
+from genai.prompt_pattern import PromptPattern
+from genai.schemas import ModelType
+from genai.utils.search_space_params import grid_search_generate_params
+
+# make sure you have a .env file under genai root with
+# GENAI_KEY=<your-genai-key>
+# GEMAI_API=<the-genai-api> (optional) DEFAULT_API = "https://workbench-api.res.ibm.com/v1"
+
+load_dotenv()
+API_KEY = os.getenv("GENAI_KEY", None)
```

Comment:
Let's add the endpoint to the example as well
```suggestion
API_KEY = os.getenv("GENAI_KEY", None)
API_URL = os.getenv("GENAI_API", None)
```